File: 0001_0100/0061/0061.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    r = 0
    for i in range(4):
        a, b = (int(x) for x in f.readline().split())
        r += a - b
    q.write("1" if r > 0 else "2" if r < 0 else "DRAW")

File: 0001_0100/0061/0061.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var a, b, r int
	for i := 0; i < 4; i++ {
		fmt.Fscan(f, &a, &b)
		r += a - b
	}
	s := "DRAW"
	if r > 0 {
		s = "1"
	} else if r < 0 {
		s = "2"
	}
	fmt.Fprint(q, s)
}
